shared/package.json

```diff
@@ -14,8 +14,8 @@
   "dependencies": {
     "@google-cloud/bigquery": "^5.0.0",
     "@google-cloud/functions-framework": "^1.3.2",
-    "@google-cloud/pubsub": "^1.1.5",
-    "@google-cloud/storage": "^4.1.3",
+    "@google-cloud/pubsub": "^2.6.0",
```

Comment:
This change is causing an issue in the metadata service call it seems.

async checkIfSubscriptionExists(topicName, subscriptionProjectId, subscriptionName)

Checking if subscription exists
(node:41179) MetadataLookupWarning: received unexpected error = Invalid response from metadata service: incorrect Metadata-Flavor header. code = UNKNOWN

Reply:
Actually, this same error occurs in master too.